students/nartowska_karina/lesson_10_organizing_files/selective_copy.py

```diff
@@ -0,0 +1,14 @@
+import shutil
+import glob
+
+
+def main():
+    ext = input('Enter an extension, e.g .txt: ')
+    new_folder_path = "new/"
```

Comment:
I think that it will be good to create in the current directory  ```new``` folder.  It will make your script will work without any changes in your scripts 😄  What do you think?